client/txn.go

```diff
@@ -273,13 +284,23 @@ func (txn *Txn) Run(b *Batch) error {
 // efficient than relying on the implicit commit performed when the transaction
 // function returns without error.
 func (txn *Txn) Commit(b *Batch) error {
-	args := &proto.EndTransactionRequest{Commit: true}
-	reply := &proto.EndTransactionResponse{}
-	b.calls = append(b.calls, proto.Call{Args: args, Reply: reply})
+	b.calls = append(b.calls, createEndCall(true))
 	b.initResult(1, 0, nil)
 	return txn.Run(b)
 }
 
+// Rollback a transaction.
+func (txn *Txn) Rollback() error {
+	return txn.send(createEndCall(false /*dont commit*/))
+}
+
+func createEndCall(commit bool) proto.Call {
```

Comment:
The similar methods in the `proto` package do not have a `Create` prefix. I would rename this `endCall`.